test/autorest/generated/headergroup/internal/headergroup/header.go

```diff
@@ -319,8 +319,8 @@ func (HeaderOperations) ResponseByteHandleResponse(resp *azcore.Response) (*Head
 	if !resp.HasStatusCode(http.StatusOK) {
 		return nil, newError(resp)
 	}
-	val := []byte(resp.Header.Get("value"))
-	return &HeaderResponseByteResponse{RawResponse: resp.Response, Value: &val}, nil
+	Value := []byte(resp.Header.Get("value"))
```

Comment:
Isn't this a base-64 encoded string that needs to be decoded?  Can be fixed separately.

Reply:
Done